Makefile

```diff
@@ -29,11 +29,20 @@ all: init
 	cd $(BUILD_DIR) && cmake $(SOURCE_PATH) -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) -DCMAKE_INSTALL_PREFIX=$(PREFIX)
 	$(MAKE) -C $(BUILD_DIR) all
 
-install: all
-	cmake -P $(BUILD_DIR)/cmake_install.cmake
+install: | uninstall all
```

Comment:
I'd just call uninstall as first command in this target:
install: all
$(MAKE) uninstall